doc/classes/ProjectSettings.xml

```diff
@@ -821,6 +821,9 @@
 		</member>
 		<member name="rendering/quality/shading/force_lambert_over_burley.mobile" type="bool" setter="" getter="" default="true">
 		</member>
+		<member name="rendering/quality/shading/gles2_high_float_precision.mobile" type="bool" setter="" getter="" default="false">
+			If [code]true[/code] and available on the target device high floating point precision for all shader computations in GLES2 is enabled.
```

Comment:
Change to ->
"If [code]true[/code] and available on the target device, enables high floating point precision for all shader computations in GLES2.
[b]Warning:[/b] High floating point precision can be extremely slow on older devices and is often not available at all. Use with caution."